src/main/kotlin/no/nav/syfo/client/norg/NorgClient.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.syfo.client.norg
+
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.plugins.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import no.nav.syfo.client.azuread.AzureAdClient
+import no.nav.syfo.client.httpClientDefault
+import no.nav.syfo.client.norg.domain.NorgEnhet
+import no.nav.syfo.tilgang.Enhet
+import no.nav.syfo.util.NAV_CALL_ID_HEADER
+import no.nav.syfo.util.callIdArgument
+import org.slf4j.LoggerFactory.getLogger
+
+class NorgClient(
+    private val azureAdClient: AzureAdClient,
+    private val baseUrl: String,
+    private val clientId: String,
+    private val httpClient: HttpClient = httpClientDefault(),
+) {
+
+    suspend fun getOverordnetEnhetListForNAVKontor(
+        callId: String,
+        enhet: Enhet
+    ): List<NorgEnhet> {
+        val url = getOverordnetEnheterForNAVKontorUrl(enhet.id)
+        try {
+            // TODO: Vet non om man kan få tom liste her, eller om norg i så fall gir feilmelding?
```

Comment:
Foresvever meg at vi har hatt noen forekomster av sånne problemer for en stund siden, men husker ikke så mye om det.

Reply:
Jeg la inn en sjekk på det 👍